openedx_webhooks/tasks/github.py

```diff
@@ -39,7 +39,25 @@ def pull_request_opened(pull_request, ignore_internal=True, check_contractor=Tru
     user = pr["user"]["login"].decode('utf-8')
     repo = pr["base"]["repo"]["full_name"]
     num = pr["number"]
-    if ignore_internal and is_internal_pull_request(pr, session=github):
+    is_internal_pr = is_internal_pull_request(pr, session=github)
+
+    if is_internal_pr and not has_internal_cover_letter(pr) and is_beta_tester_pull_request(pr, session=github):
+        logger.info(
+            "Adding cover letter template to PR #{num} against {repo}".format(
+                repo=repo, num=num,
+            ),
+        )
+        comment = {
+            "body": github_internal_cover_letter(pr),
+        }
+        url = "/repos/{repo}/issues/{num}/comments".format(
```

Comment:
Nope, pull requests have URLs that look like `.../pulls/42`, while issues have URLs that look like `.../issues/42`.